account_move_substate/models/account_move.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Ecosoft (<http://ecosoft.co.th>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class BaseSubstateType(models.Model):
+    _inherit = "base.substate.type"
+
+    model = fields.Selection(
+        selection_add=[("account.move", "Account Move")],
+        ondelete={"account.move": "cascade"},
+    )
+
+
+class AccountMove(models.Model):
+    _inherit = ["account.move", "base.substate.mixin"]
+    _name = "account.move"
+    _state_field = "state"
+
+    def _track_template(self, changes):
```

Comment:
This function will add on base_substate. Can you delete this please
https://github.com/OCA/server-ux/pull/736

Reply:
@bosd can you take a look here?

Reply:
Will do, Let's first get 736 merged. As this one depends on it.

Reply:
@Saran440 Can you re-review? 🙏 
Please make it non blocking. As your proposed refactor is not included yet for V16.

There is still ongoing discussion on the refactor. It should not block this PR.
We can always refactor this module, once the proposed changes are in.

Reply:
Just noticed this is non blocking as well